apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/logs/_components/DocumentLogs/DocumentLogInfo/Evaluations.tsx

```diff
@@ -0,0 +1,104 @@
+import {
+  EvaluationMetadataType,
+  EvaluationResultableType,
+} from '@latitude-data/core/browser'
+import { ResultWithEvaluation } from '@latitude-data/core/repositories'
+import {
+  Button,
+  Icon,
+  Text,
+  useCurrentCommit,
+  useCurrentProject,
+} from '@latitude-data/web-ui'
+import {
+  evaluationMetadataTypes,
+  evaluationResultTypes,
+} from '$/app/(private)/evaluations/_components/ActiveEvaluations/Table'
+import { useCurrentDocument } from '$/app/providers/DocumentProvider'
+import { ROUTES } from '$/services/routes'
+import Link from 'next/link'
+
+import { MetadataItem } from '../../../../_components/MetadataItem'
+import { ResultCellContent } from '../../../../evaluations/[evaluationId]/_components/EvaluationResults/EvaluationResultsTable'
+
+function EvaluationResultItem({ result, evaluation }: ResultWithEvaluation) {
+  if (result.resultableType === EvaluationResultableType.Text) {
+    return (
+      <Text.H5 align='left' color='foregroundMuted'>
+        {result.result || '-'}
+      </Text.H5>
+    )
+  }
+
+  return <ResultCellContent evaluation={evaluation} value={result.result} />
+}
+
+function evaluationResultLink({ result, evaluation }: ResultWithEvaluation) {
+  const { project } = useCurrentProject()
+  const { commit } = useCurrentCommit()
+  const document = useCurrentDocument()
+
+  const route = ROUTES.projects
+    .detail({ id: project.id })
+    .commits.detail({ uuid: commit.uuid })
+    .documents.detail({ uuid: document.documentUuid })
+    .evaluations.detail(evaluation.id).root
+
+  if (evaluation.metadataType === EvaluationMetadataType.Manual) {
+    return `${route}?documentLogId=${result.documentLogId}`
+  }
+
+  return `${route}?resultUuid=${result.uuid}`
+}
+
+export function DocumentLogEvaluations({
+  evaluationResults = [],
+}: {
+  evaluationResults?: ResultWithEvaluation[]
+}) {
+  if (!evaluationResults.length) {
+    return (
+      <Text.H5 color='foregroundMuted' centered>
+        There are no evaluation results for this log
+      </Text.H5>
+    )
+  }
```

Comment:
How looks this case?

Reply:
![image](https://github.com/user-attachments/assets/b6f16143-3db7-4973-bcce-03f429e6602e)